.github/workflows/release.yml

```diff
@@ -59,3 +59,13 @@ jobs:
       - name: Copy to latest
         if: env.pre_release == 'false'
         run: gsutil -m cp "gs://runme-artifacts-41eac6/${{ env.version }}/*" gs://runme-artifacts-41eac6/latest
+      - name: Bump Homebrew Formula
+        uses: mislav/bump-homebrew-formula-action@v2
+        # skip prereleases
+        if: ${{ !contains(github.ref, '-') }}
+        with:
+          formula-name: runme
+          homebrew-tap: Homebrew/homebrew-core
+          base-branch: master
```

Comment:
```suggestion
          base-branch: main
```

Reply:
This refers to https://github.com/Homebrew/homebrew-core main branch afaik. I guess we could also just remove this as it is the default value I assume